Source/WebKit/Configurations/NetworkingExtension.xcconfig

```diff
@@ -26,3 +26,6 @@
 INFOPLIST_FILE = Shared/AuxiliaryProcessExtensions/NetworkingExtension-Info.plist;
 INFOPLIST_KEY_CFBundleDisplayName = NetworkingExtension;
 PRODUCT_BUNDLE_IDENTIFIER = com.apple.WebKit.NetworkingExtension;
+PRODUCT_BUNDLE_EXECUTABLE = NetworkingExtension;
+PRODUCT_BUNDLE_NAME = NetworkingExtension;
```

Comment:
It's too bad we have to have so much boiler plate. It'd be nice if all the duplicates could be handled with a variable replacement.

Reply:
That is a good point. I can look into that in a follow-up patch.

Thanks for reviewing!